src/Microsoft.Azure.WebJobs.Host/Common/OutArgumentBinding/CommonAsyncCollectorValueProvider.cs

```diff
@@ -6,20 +6,20 @@
 using System.Threading.Tasks;
 using Microsoft.Azure.WebJobs.Host.Bindings;
 
-namespace Microsoft.Azure.WebJobs.ServiceBus
+namespace Microsoft.Azure.WebJobs.Host
 {
     // TUser is the parameter type from the user function we're binding to. 
     // TMessage is from the underying IAsyncCollector<TMessage>
     internal class CommonAsyncCollectorValueProvider<TUser, TMessage> : IOrderedValueBinder
```

Comment:
I'm wondering why you need the "Common" prefix on all these types? That seems like a vestige of them living in a Common folder? Recommend removing it across all the types since it doesn't add anything.